turbinia/workers/bulk_extractor.py

```diff
@@ -72,6 +72,24 @@ def run(self, evidence, result):
       return result
     return result
 
+  def check_xml_attrib(self, xml_key):
```

Comment:
Just a tiny nit here:  The first line should end with a period.  I need to add pylint to the automated tests, but maybe run pylint manually on these files just to check?  Thanks!

Reply:
Fixed!